data/store.json

```diff
@@ -439,6 +439,118 @@
             }
         ]
     },
+    {
+        "id": 11,
+        "printful_id": 162,
+        "name": "Men's Purple Vala Tee",
+        "description": "Hand-lettered Vala design in lavender and white on a lightweight purple tri-blend tee by Bella + Canvas. <br/><a class='read-more' href='https://www.bellacanvas.com/fit-guide'>Size Chart</a>",
+        "image": "images\/store\/apparel\/mens_tee_vala_purple.png",
+        "type": "Apparel",
+        "variants": [
+            {
+                "id": 1,
+                "printful_id": 6505,
```

Comment:
This is for the black tee. The series for the purple tee starts at `6569`

Reply:
Working on it :sob: